### ISSUE TITLE
Improve documentation for setup of AGE extension

### ISSUE THREAD

**Issue by Johan511** [apache/age#311]
While trying to setup the extension, I did face a multitude of problems mostly due to the fact that I already had pg14 installed on my system and had to try to figure out on my own how to install pg11, change the pg_config file, and make sure I connect to the pg11 database using psql.

I hadn't found any proper resources on the internet on how to use multiple versions postgres on the same system without containers.

I would like to add to the documentation the steps to properly setup the age extension when having multiple versions of postgres, and request any regular contributors to guide me on how I could add to the documentation.

**Comment by Johan511** [comment]
Is there anyway I can contribute to the documentation ?

**Comment by dehowef** [comment]
@Johan511 yes! you can clone https://github.com/apache/age-website and make changes to the documentation in https://github.com/apache/age-website/tree/master/docs

Once you have made the desired changes, you may issue a pull request and someone will review the changes.

**Comment by jccampagne** [comment]
Yesterday I had a similar problem relating to mis-aligned versions of Age and Postgres
https://github.com/apache/age/issues/1991

Indeed, it would be nice to have a note specifically on aligning the versions.

**Comment by Johan511** [comment]
Hi @jccampagne, I had updated the documentation to explain a fix for this issue. You can have a look at the [PR](https://github.com/apache/age-website/pull/85/files) over here

**Comment by jccampagne** [comment]
@Johan511 Thanks for pointing this out - although I did read this bit on the website, it was not clear at first that I had to align the versions of Apache Age and PostgreSQL.
It's evident in hindsight. But I thought it would be good to spell it out for the new users to avoid running in the same issue as me...

https://github.com/apache/age-website/pull/332/files

I also see that tags are used for release candidates (e.g. `PG16/v1.5.0-rc0`) and branches for versions (`release/PG16/1.5.0`). Is that correct? Are tags not used for releases?

**Comment by jrgemignani** [comment]
@jccampagne When we do releases, we update the specific branch to the new version, via a PR for that branch. We then create a new branch `release/<branch name>/<version>` from the specific branch. The release tag is then made from that new branch as are all of the artifacts and and zip'd files.

We could probably improve and streamline the process, but this has been how we've done releases.

**Comment by jrgemignani** [comment]
@jccampagne And yes, the versions of PG and AGE need to be the same, hence the branch names PG<version> :)
